omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -248,6 +248,11 @@ Start the PostgreSQL server.
     $ brew services start postgresql
     $ pg_ctl -D /usr/local/var/postgres/ -l /usr/local/var/postgres/server.log start
 
+You will need to install ``brew services`` first:
+
+::
+     $ brew tap gapple/services
```

Comment:
I rather suspect this is too far indented - looks further in than the points above to me.